### DISCUSSION TITLE
Struggling with git&github using MAC!!!!

### DISCUSSION THREAD

**Discussion by ihkim8** [community/community#41730] [New to GitHub]
Hihi 
Just started using github. It's a day 2. 
On the first day, I used Windows. It was difficult but okay.
But today, I'm using my mac and git&github is not working! 

So far, I've managed to install git. But, git file doesn't appear on my vs code. I'm trying to work it out with adding a new SSH key. But I can't find SSH key.. 

I just have no clue where to start. Don't know from where I should fix. 
I'd really appreciate your help on teaching me how to make git&github work on mac. 

Cheers!

**Comment by sagarbangade** [comment]
[https://stackoverflow.com/questions/52522565/git-is-not-working-after-macos-update-xcrun-error-invalid-active-developer-pa](url)

**Reply by ihkim8** [reply]
Hihi thanks for this! Unfortunately, I can't view the link :S

**Reply by sagarbangade** [reply]
Always welcome my friend ☺️
